cmd/marker/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"time"
+
+	"github.com/golang/glog"
+	"kubevirt.io/ovs-cni/pkg/marker"
+)
+
+func main() {
+	flag.Parse()
+
+	nodeName := os.Getenv("NODE_NAME")
+
+	for {
+		err := marker.Update(nodeName)
+		if err != nil {
+			glog.Errorf("Update failed: %v", err)
+		}
+		time.Sleep(10 * time.Second)
```

Comment:
perhaps it would make sense to pass this constant as a cmdline parameter.

Reply:
Fixed